app/views/layouts/application.html.erb

```diff
@@ -39,6 +39,13 @@
   </div>
 
   <%= yield %>
+    <% if Rails.env.development? %>
+      <script>
+        window.ga = function() {
+          console.log(arguments);
```

Comment:
Can this check for the existence of console.log? Otherwise this will never work in IE in development, which we should be checking when making front end changes.